docs/user_guide/capsules/isolation.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: documentation
+title: Capsule Isolation
+sidebar: sidebars/documentation.html
+---
+
+# Capsule Isolation
+
+The goal of Capsule Isolation is to provide a single endpoint for all of a client's communication, so that in remote network segments, you need only open Firewall ports to the Capsule itself.
+
+![Diagram of Communication](./isolation.png)
+
+## Content and Configuration Services
+
+There are 4 primary areas that require client communication:
+
+### Package Installation
+
+That is, yum.  Katello Capsules by default have the Pulp feature, which mirrors content for the selected Lifecycle Environments.  Enable the Content Mirror with `--pulp=true` on the capsule-installer. Clients need to be able to communicate with the Capsule on port 443/tcp.
+
+### Katello Agent
+
+The Katello agent is a goferd plugin that allows you to schedule remote actions on hosts such as package installation, updates, etc.  In order to use the Katello agent against a capsule, you will need to enable the Qpid Dispatch Router feature (`--qpid-router=true`) in the capsule-installer.  Clients need to be able to communicate with the Capsule on port 5467/tcp.
```

Comment:
This will be enabled by default ? Might be worth mentioning in all cases that the behavior is default.